### PR TITLE
docs: Wrap samples with future prefix

### PR DESCRIPTION
**Pull request by Strykrol**
Standardizing Security Command Center samples to use 'securitycenter' prefixing. 

- Wrapped existing samples to keep published doclinks unbroken
- Replace region tags that aren't published. 

Once this PR is through, published sample inclusions will be updated to use the new prefix, then I'll come through again and remove the unused block wraps.

### CODE REVIEW

File: securitycenter/src/receive_notification.php [hunk 1]
```diff
@@ -22,7 +22,7 @@
 }
 list($_, $projectId, $subscriptionId) = $argv;
 
-// [START scc_receive_notifications]
+// [START securitycenter_receive_notifications]
```
Comment (bshaffer) [inline securitycenter/src/receive_notification.php:25]: Curious why this one was removed while the others were wrapped, but otherwise LGTM

**Review by bshaffer — APPROVED** [review]

File: securitycenter/src/receive_notification.php (hunk 1 above)
Comment (Strykrol) [reply securitycenter/src/receive_notification.php:25]: This PHP sample is not published live in any docs, so fully changing this should be no-harm-no-foul.

File: securitycenter/src/list_notification.php [hunk 2]
```diff
@@ -22,6 +22,7 @@
 }
 list($_, $organizationId) = $argv;
 
+// [START securitycenter_list_notification_configs]
```
Comment (bshaffer) [inline securitycenter/src/list_notification.php:25]: The tests are failing because there's trailing whitespace at the end of this line

Comment (Strykrol) [reply securitycenter/src/list_notification.php]: Done, thank you!

File: securitycenter/src/create_notification.php [hunk 3]
```diff
@@ -22,6 +22,7 @@
 }
 list($_, $organizationId, $notificationConfigId, $projectId, $topicName) = $argv;
 
+// [START securitycenter_ create_notification_config]
```
Comment (tmatsuo) [inline securitycenter/src/create_notification.php:25]: `[START securitycenter_ create_notification_config]`

There is a unnecessary space in the region tag part. Please fix.

File: securitycenter/src/create_notification.php [hunk 4]
```diff
@@ -51,3 +52,4 @@
 printf('Notification config was created: %s' . PHP_EOL, $response->getName());
 
 // [END scc_create_notification_config]
+// [END securitycenter_ create_notification_config]
```
Comment (tmatsuo) [inline securitycenter/src/create_notification.php:55]: ditto

**Review by tmatsuo — CHANGES_REQUESTED** [review]

File: securitycenter/src/create_notification.php (hunk 4 above)
Comment (Strykrol) [reply securitycenter/src/create_notification.php]: Thank you!

File: securitycenter/src/create_notification.php (hunk 3 above)
Comment (Strykrol) [reply securitycenter/src/create_notification.php]: Good catch, thanks.

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**